tests/unit/vdom.ts

```diff
@@ -2708,4 +2709,38 @@ describe('vdom', () => {
 			});
 		});
 	});
+
+	describe('behavior', () => {
+		it('will append nodes with attributes already attached', (test) => {
```

Comment:
This tests attributes, how about properties too?

Reply:
The intention of the test is that `setProperties` was called before `attachNode`. I wanted to avoid describing internal functions in the test. Would it be more clear with another name?